packages/next/server/app-render.tsx

```diff
@@ -180,13 +180,20 @@ function createServerComponentRenderer(
     globalThis.__next_chunk_load__ = () => Promise.resolve()
   }
 
-  const writable = transformStream.writable
-  const ServerComponentWrapper = (props: any) => {
-    const reqStream: ReadableStream<Uint8Array> = renderToReadableStream(
-      <ComponentToRender {...props} />,
-      serverComponentManifest
-    )
+  let RSCStream: ReadableStream<Uint8Array>
```

Comment:
Any reason this is lazy, can’t it just be created here?

Reply:
With the fetch approach especially it’s best to start the fetch as early as possible since every cpu cycle spent delaying it will show up in the overall render time.